### ISSUE TITLE
Timetable order by Timetable Order field

### ISSUE THREAD

**Issue by ekeaton** [Trillium-Solutions/transit-custom-posts#119]
According to the README

> Vertically reorder timetables
Timetables are sorted by the 'timetable id’ by default. To reorder timetables vertically, set the route to 'Order by': 'Timetable Order Field' and set the timetables 'Timetable Order' field in the numerical order you want the timetables to appear.

This feature is a bit confusing on how to order the Timetable Order fields. It looks like it will accept any value from 1 to x for ordering. Originally it seemed it only ordered by 1 and 2. For example, if Saturday is before Weekday after running TCP and you want to reverse them. Then you set Weekday to 1 and Saturday to 2 to switch the labels. Similarly the same for Outbound and Inbound they are 1 and 2. 

However, when trying this feature out on LRTA this observation was made: Weekday Outbound 1 Saturday Outbound 2 Weekday Outbound 3 Saturday Inbound 4 in order for them all to appear and it was a guessing game. Also when dealing with many routes, swtiching out each timetable can be tendious. 

Ideally a future update for this feature: Order the timetables all in one section (on the settings page), instead of Route per route and Timetable per timetable for Timetable re-ordering of daygroups and directions.

**Comment by danielleogden** [comment]
@chancecorbeil @ekeaton I was poking around and found this closed ticket: https://github.com/Trillium-Solutions/transit-custom-posts/issues/7

It's hard for me to tell what was going on here. Possibly they were planning on managing timetable sequence within gtfs manager? What do you make of this Estelle?

**Comment by ekeaton** [comment]
Yeah, It looks like Brendan was telling them it's already possible with g2h IF they use the timetable_sequence field in timetables.txt and then also timetable pages. But I don't know if this was actually added to GTFSM.

@evantrillium Does your team have the option to use timetable_sequence in timetables.txt now? RE: https://github.com/Trillium-Solutions/transit-custom-posts/issues/7

**Comment by evantrillium** [comment]
we do not - it is not included in timetables.txt nor in GTFSM
